### ISSUE TITLE
Squashed migrations

### ISSUE THREAD

**Issue by sximba** [etianen/django-reversion#699]
Will the squashed migrations (i.e 0001, 0002, 0003, 0004) ever be deleted as advised in the django docs?

The squashed migration has existed for 2 years now. I think it is safe to assume everyone who uses the project has applied it.

Their presence in the code base makes it impossible to unapply migrations and leave the django migration tree clean.

**Comment by etianen** [comment]
Good point. This has been done in master.

**Comment by BallisticPain** [comment]
Yet we're still having issues where people are NOT ready for them to be gone. I've copied them into a separate directory myself... :( Having to handle it manually... Oh well I'll be good after this... :-D

**Comment by luzfcb** [comment]
I was bitten by it. Will be included in the changelog in which version was removed.

**Comment by etianen** [comment]
Okay... I won't be removing squashed migrations any more. I'll even take a pull request to put them back in.

**Comment by merwok** [comment]
Can confirm than doing a big update on a existing live project after two years does happen.  Current migration is 0002_auto_20141216_1509 :)

**Comment by etianen** [comment]
**I'll even take a pull request to put them back in.**

**Comment by libcthorne** [comment]
I hit the same problem. PR here, as suggested: #757

**Comment by etianen** [comment]
Pull request merged. If people could confirm that it's working for them on master, I can make this into a release. Thanks @libcthorne

**Comment by libcthorne** [comment]
I can confirm `master` works for me:
```
Running migrations:
  ...
  Applying reversion.0003_auto_20160601_1600... OK
  Applying reversion.0004_auto_20160611_1202... OK
```
A release would be appreciated.

**Comment by etianen** [comment]
Released 3.0.1

**Comment by etianen** [comment]
This has broken the upgrade from v3.0.0. It looks like it's impossible to add squashed revisions back in without triggering some broken behavior in the django migrations framework. See #763
